src/TextInput/TextInput.tsx

```diff
@@ -158,12 +157,13 @@ const Content = styled.div<IInputOutline>`
 
 const Input = styled.input<IInput>`
   border: none;
+  background-color: transparent;
```

Comment:
An `input` has a white background by default so I removed it as it was clashing with the border